index.html

```diff
@@ -141,17 +141,17 @@ <h2>Attributes</h2>
           class="idlAttrType">double</span>, nullable</dt>
         <dd>
           <p>This attribute allows the application to give a hint to the User
-          Agent about what playout delay is acceptable. The User Agent SHOULD
-          NOT playout audio or video that is received unless this amount of time
-          has passed in seconds, allowing the User Agent to perform more or less
-          buffering than it might otherwise do. This allows to influence the
-          tradeoffs between having a higher delay and the risk that buffers such
-          as the jitter buffer will run out of audio or video frames to play due
-          to network jitter. This API only serves to provide a hint, and the
-          User Agent may ignore it. The hint SHOULD NOT be followed if it
-          significantly impacts audio or video quality (e.g. bad network), or if
-          the value implies allocating larger buffers than the User Agent is
-          willing to provide.</p>
+          Agent about what total delay is acceptable before the playout. The
+          User Agent SHOULD NOT playout audio or video that is received unless
+          this total amount of time has passed in seconds, allowing the User
+          Agent to perform more or less buffering than it might otherwise do.
+          This allows to influence the tradeoffs between having a higher delay
+          and the risk that buffers such as the jitter buffer will run out of
+          audio or video frames to play due to network jitter. This API only
+          serves to provide a hint, and the User Agent may ignore it. The hint
+          SHOULD NOT be followed if it significantly impacts audio or video
+          quality (e.g. bad network), or if the value implies allocating larger
+          buffers than the User Agent is willing to provide.</p>
```

Comment:
I wonder if we can more specifically say that user agents must define a _floor value_ and a _ceiling value_ to represent these properties.

That way we can put normative requirements on that user agents MUST respect values in between those self-defined limits. That seems like the right trade-off for web compat to me.

Update: I see the prose steps below sort of have these concepts already, but maybe we can formalize it as definitions?